SDWebImage/UIView+WebCacheOperation.h

```diff
@@ -16,6 +16,14 @@
 // All the stored operations are weak, so it will be dalloced after image loading finished. If you need to store operations, use your own class to keep a strong reference for them.
 @interface UIView (WebCacheOperation)
 
+/**
+ *  Get the image load operation for key
+ *
+ *  @param key key for identifying the operations
+ *  @return the image load operation
+ */
+- (nullable id<SDWebImageOperation>)sd_imageLoadOperationForKey:(nullable NSString *)key;
```

Comment:
This is actually an advanced ability to allow user to get current loading operation for UIView category. See the use demo below in comments.